keps/sig-network/4444-service-traffic-distribution/README.md

```diff
@@ -311,14 +324,29 @@ the value configured for `trafficDistribution`
 * This leverages existing implementation, requiring no major changes.
 
 #### `PreferClose`
-* **Meaning:** Attempts to route traffic to endpoints within the same zone as
-  the client. If no endpoints are available within the zone, traffic would be
-  routed to other zones.
+* **Meaning:**
+  * Attempts to route traffic to endpoints within the same zone as the client. If no
+    endpoints are available within the zone, traffic would be routed to other zones.
+  * If the Service has endpoints that are part of a DaemonSet deployed to all nodes,
+    then it attempts to route traffic to endpoints on the same node as the client. If
+    no endpoints are available on the same node, traffic would be routed to other
+    nodes (though still preferring the same zone, if relevant).
+
 * This preference will be implemented by the use of Hints within EndpointSlices.
-* We already use Hints to implement `service.kubernetes.io/topology-mode: Auto`
-  In a similar manner, the EndpointSlice controller will now also populate hints
-  for `trafficDistribution: PreferClose` -- although in this case, the zone hint will
-  match the endpoint of the zone itself.
+  * We already use `Hints.ForZones` to implement
+    `service.kubernetes.io/topology-mode: Auto` In a similar manner, the
+    EndpointSlice controller will now also populate zone hints for
+    `trafficDistribution: PreferClose` -- although in this case, the zone hint will
```

Comment:
Maybe a different name like "PreferCloser", "PreferNode", "PreferLocal" would help simplify the overall implementation